src/runtime/Makefile

```diff
@@ -164,6 +164,9 @@ DEFDISABLEGUESTSECCOMP := true
 #Default experimental features enabled
 DEFAULTEXPFEATURES := []
 
+#Default sandbox param
+DEFSANDBOXPARAM := [on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny]
```

Comment:
I think the square brackets `[]` are not needed.

```
DEFSANDBOXPARAM := on,obsolete=deny,elevateprivileges=deny,spawn=deny,resourcecontrol=deny
```

In addition, you need to add `DEFSANDBOXPARAM` to `USER_VAR` as follows in order to replace the variable in generated configuration files.

```
USER_VARS += DEFSANDBOXPARAM
```

Ref: https://github.com/kata-containers/kata-containers/blob/main/src/runtime/Makefile#L437

Reply:
Thank you @ManaSugi , I've updated the code according to your comments.